README.md

```diff
@@ -3,7 +3,21 @@
 
 Copyright © 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All Rights Reserved. SPDX-License-Identifier: Apache-2.0
 
-## Requirements
+# Daml Finance
+
+Welcome to Daml Finance, a collection of purpose-built libraries to enable rapid development of enterprise-grade tokenization solutions.
+
+## Why do I need it?
+
+Implementing basic financial concepts like ownership or economic terms of an asset is a complex and tedious task. By providing common building blocks, Daml Finance increases delivery velocity and shortens the time-to-market when developing Daml applications.
+
+## Getting started using the library
+
+The [Daml Finance documentation](https://digital-asset.github.io/daml-finance/) provides a number of options to get started using the library.
+
+## Contributing to Daml Finance
```

Comment:
Open point: shall we specify that we are currently not accepting external contributions?

Reply:
Don't think we need to